### PR TITLE
Automation Demos Json

### PR DESCRIPTION
**Pull request by plamen-peev**
This commit is related to point 2 at:
https://github.com/eclipse/smarthome/issues/2220

### CODE REVIEW

File: bundles/automation/org.eclipse.smarthome.automation.sample.json.demo/ESH-INF/automation/rules/DemoRule.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            "configuration":{  
+               "itemName":"DemoSwitch"
+            }
+         }
+      ],
+      "actions":[  
+         {  
+            "id":"RuleAction",
+            "label":"Post command to an item",
+            "description":"posts commands on items",
```
Comment (danchom) [inline bundles/automation/org.eclipse.smarthome.automation.sample.json.demo/ESH-INF/automation/rules/DemoRule.json]: Description sentence should start with upper case.

**Comment by danchom** [comment]
LGTM

File: bundles/automation/org.eclipse.smarthome.automation.sample.java.demo/src/main/java/org/eclipse/smarthome/automation/sample/java/demo/SampleJavaDemo.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2016 by ProSyst Software GmbH
+ * http://www.prosyst.com
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information
```
Comment (kaikreuzer) [inline bundles/automation/org.eclipse.smarthome.automation.sample.java.demo/src/main/java/org/eclipse/smarthome/automation/sample/java/demo/SampleJavaDemo.java]: please add EPL license header - I hope this code isn't confidential ;-)

File: bundles/automation/pom.xml [hunk 3]
```diff
@@ -37,6 +37,7 @@
     <module>org.eclipse.smarthome.automation.sample.extension.java</module>
     <module>org.eclipse.smarthome.automation.rest</module>
     <module>org.eclipse.smarthome.automation.sample.rest.api</module>
+	<module>org.eclipse.smarthome.automation.sample.json.demo</module>
```
Comment (kaikreuzer) [inline bundles/automation/pom.xml]: please replace tab by spaces here

**Comment by kaikreuzer** [comment]
@plamen-peev Could you please rebase your branch? We are good to merge it then, thanks!

**Pull request merged**